### DISCUSSION TITLE
Click on Map or write a text address

### DISCUSSION THREAD

**Discussion by bgmrsln** [bounswe/bounswe2023group2#60] [Q&A] [answered]
We get the location data in various occations such as resource entry, event creation, user profile, need creation, action. Will it be like click on a map icon, a map will be opened and click on the location? There should also be a search button in the map. I am not sure how it would affect the implementation. If location will be chosen on the map I think these requirements should be added to map visualization and user actions?

**Comment by mehmetkuzu** [comment]
Actually, in profile page mockup I could not add the map. Trying to save the mockup with map, I got "drawing is too large 1MB."
Me too, I also think that address data should also be collected using a map. 
(But the users should also be given the opportunity to enter text format address information. For nearly every case.)

**Reply by bgmrsln** [reply]
For the map, there is a map icon on the left that can be used

**Comment by bgmrsln** [comment]
As an example someone can enter themselves as a resource. Say they are a driver. We dont need to know where exactly this person is. A higher level address would be enough.

**Reply by mehmetkuzu** [reply]
So, any text is also ok to begin with?
(Like Antakya Armutlu girişi)

**Comment by mehmetkuzu** [comment, marked as answer]
I saved the profile pages in another mockup file. Please check that. I used a dummy map in diagrams.net shapes.
